src/js/view/controlbar.js

```diff
@@ -15,7 +16,9 @@ define([
         element.style.display = 'none';
 
         if (click) {
-            element.onclick = function() { click(); };
+            // Some actions receive parameters which change functionality based on their presence (E.G. play)
```

Comment:
Will we need this comment if we rename the variable from "click" to "apiAction"?